.changeset/itchy-apes-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Improved accessibility of `Button` and `Link` components on Windows-based high contrast themes. Outlines are now visible only when focused, and border underlines will appear correctly.
```

Comment:
@stamat I've added a changelog entry for this update ☝️, which hopefully summarizes what these changes do. Could you please check its accurate on your end, and that I haven't missed anything?